File: counting.java
public class counting {
    public static void main(String[] args) {
        int count = 0;
        long startTime = System.currentTimeMillis();
        for (int i = 1; i <= 1000000000; i++) {
            count++;
        }
        long endTime = System.currentTimeMillis();
        System.out.println("Count: " + count);
        System.out.println("Execution time: " + (double)(endTime - startTime) / 1000 + " seconds");
    }
}

File: counting.c
#include <stdio.h>
#include <time.h>

int main() {
    int count = 0;
    clock_t start = clock();
    for (int i = 1; i <= 1000000000; i++) {
        count++;
    }
    clock_t end = clock();
    printf("Count: %d\n", count);
    printf("Execution time: %f seconds\n", (double)(end - start) / CLOCKS_PER_SEC);
    return 0;
}

